execution/wasm/wasm.go

```diff
@@ -73,6 +74,35 @@ func (e *execContext) ResolveFunc(module, field string) exec.FunctionImport {
 	}
 
 	switch field {
+	case "call":
+		return func(vm *exec.VirtualMachine) int64 {
+			// gas := int(uint64(vm.GetCurrentFrame().Locals[0]))
```

Comment:
Ideally the call function should be implemented properly and then the precompiles in https://github.com/hyperledger/burrow/blob/master/execution/native/precompiles.go can be called.